crates/lang/src/reflect/trait_def/info.rs

```diff
@@ -152,3 +152,6 @@ pub trait TraitModulePath {
     /// This is just for convenience.
     const NAME: &'static str;
 }
+
+/// Used to identify ink! structs.
+pub trait InkStruct {}
```

Comment:
So I get what you're trying to do with this, but it really isn't used to identify ink! structs. It's really just a marker for `CallBuilder`.

If we're going to stick with the blanket impl approach I'd change this to be something like:

```rust
pub trait CallBuilder: ContractEnv + ToAccountId {}
```

Reply:
Yea, that is good idea=)